xray/services/xsc/xsc.go

```diff
@@ -0,0 +1,52 @@
+package xsc
```

Comment:
I think we should add a comment explaining the difference between XscInnerService and XscServicesManager. It's a bit confusing

Reply:
added comment